execution/scheduler.go

```diff
@@ -243,23 +239,17 @@ func (e *Scheduler) emitVUsAndVUsMax(ctx context.Context, out chan<- metrics.Sam
 				emitMetrics()
 			case <-ctx.Done():
 				return
-			case <-e.stopVUsEmission:
-				return
 			}
 		}
 	}()
+
+	return wg.Wait
 }
 
-// Init concurrently initializes all of the planned VUs and then sequentially
-// initializes all of the configured executors.
-func (e *Scheduler) Init(ctx context.Context, samplesOut chan<- metrics.SampleContainer) (err error) {
-	e.emitVUsAndVUsMax(ctx, samplesOut)
-	defer func() {
-		if err != nil {
-			close(e.stopVUsEmission)
-			<-e.vusEmissionStopped
-		}
-	}()
+// initVusAndExecutors concurrently initializes all of the planned VUs and then
+// sequentially initializes all of the configured executors.
+func (e *Scheduler) initVusAndExecutors(ctx context.Context, samplesOut chan<- metrics.SampleContainer) (err error) {
```

Comment:
nit: 
```suggestion
func (e *Scheduler) initVUsAndExecutors(ctx context.Context, samplesOut chan<- metrics.SampleContainer) (err error) {
```

Reply:
Resolved by [this force-push](https://github.com/grafana/k6/compare/60c1026af3fd2691487d52f313d7edea99c1b8bd..10700cd485373867c65d1e2e5ea654d3d7fa6fce).